lib/src/flutter_list_view_render.dart

```diff
@@ -312,7 +312,7 @@ class FlutterListViewRender extends RenderSliver
       var currentOffset = scrollOffset + compensationScroll;
       var targetOffsetFromTop = elementOffset - currentOffset;
       var distance = targetOffsetFromTop - _jumpDistanceFromTop;
-      if (distance != 0.0) {
+      if (distance < 0.0) {

```

Comment:
It is incorrect, The compensation distance may positive, because it may jump to before or after.

Reply:
change code like 
```
if (_jumpToElement != null) {
      var elementOffset = _jumpToElement!.offset;
      var currentOffset = scrollOffset + compensationScroll;
      var targetOffsetFromTop = elementOffset - currentOffset;
      var distance = targetOffsetFromTop - _jumpDistanceFromTop;
      if (distance != 0.0) {
        compensationScroll += distance;
      }
      if (scrollOffset + compensationScroll < 0) {
        compensationScroll = -scrollOffset;
      } else if (childManager.totalItemHeight -
              scrollOffset -
              compensationScroll <
          viewportHeight) {
        compensationScroll =
            childManager.totalItemHeight - viewportHeight - scrollOffset;
        if (scrollOffset + compensationScroll < 0) {
          compensationScroll = 0;
        }
      } else if (scrollOffset + compensationScroll >
          childManager.totalItemHeight) {
        compensationScroll = childManager.totalItemHeight - scrollOffset;
      }
    }
```